build/dev/probe_local_instance.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+echo "Re/Create probe_local_instance_test_result file"
+RESULTFILE="$(pwd)/probe_local_instance_test_result"
+touch "${RESULTFILE}"
+cat /dev/null > "${RESULTFILE}"
+declare -a localhost_containers=("dss_sandbox_local-dss-http-gateway_1" 
```

Comment:
The issue on the `for` loop starts here -- this command only sets `localhost_containers` to "`dss_sandbox_local-dss-http-gateway_1`" (check with `echo $localhost_containers` just before the `for` loop).  Replacing this set of lines with `localhost_containers="dss_sandbox_local-dss-http-gateway_1 dss_sandbox_local-dss-grpc-backend_1 dss_sandbox_local-dss-crdb_1"` seems to resolve the issue, but I'm not sure how to split strings across multiple lines in bash.